bedrock/utils/management/command_base.py

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from django.core.management.base import BaseCommand
+
+from sentry_sdk import capture_exception
+
+
+class SentryAlertingBaseCommand(BaseCommand):
+    def handle(self, *args, **options):
+        """Custom wrapper for management commands that failures
+        anywhere in the main handle() call get to Sentry.
+
+        self._handle is an expected method of any subclass of SentryAlertingBaseCommand.
+
+        Usage: in your management command, rename/create the handle() method so that
+        it is _handle()
+        """
+        try:
+            self._handle(*args, **options)
```

Comment:
Another option, if you don't want to have to change the name of all of the `handle()` methods, is to override the `execute()` method instead, which is the method in the `BaseCommand` class that calls `self.handle()`. Not required, just another idea.